### PR TITLE
chore(worker): tag latency per worker

### PR DESCRIPTION
**Pull request by sileht**
Change-Id: Id9c9fc4487e98776ce39c1d14f80b3bb2049a994

### CODE REVIEW

File: mergify_engine/worker.py [hunk 1]
```diff
@@ -1011,12 +1012,13 @@ async def _stream_worker_task(self, stream_processor: StreamProcessor) -> None:
...
-                    await stream_processor.consume(
-                        bucket_org_key, owner_id, owner_login_for_tracing
-                    )
+                with sentry_sdk.Hub(sentry_sdk.Hub.current) as hub:
+                    with hub.configure_scope() as scope:
+                        scope.set_tag("gh_owner", owner_login_for_tracing)
+                        scope.set_user({"username": owner_login_for_tracing})
+                        await stream_processor.consume(
+                            bucket_org_key, owner_id, owner_login_for_tracing
+                        )
```
Comment (jd) [inline mergify_engine/worker.py:1021]: unrelated?

**Comment by sileht** [comment]
@mergifyio rebase

**Review by jd — APPROVED** [review]
